client/src/components/user-admin/UserManagement.js

```diff
@@ -42,69 +49,108 @@ const UserManagement = ({ users, setUserToEdit }) => {
+            display: 'flex',
+            flexDirection: 'row',
+            justifyContent: 'center',
+            whiteSpace: 'nowrap',
+            mx: '0.5rem'
+            }}>
+              <Button
+                sx={Buttonsx}
+                type="button"
+                variant={
+                  searchResultType === 'name'
+                  ? 'contained'
+                  : 'secondary'
+                }
+                onClick={buttonSwap}
+              >
+                Results by Name
+              </Button>
+              <Button
+                sx={Buttonsx}
+                type="button"
+                variant={
+                  searchResultType === 'email'
+                  ? 'contained'
+                  : 'secondary'
+                }
+                onClick={buttonSwap}
+              >
+                Results by Email
+              </Button>
+          </ButtonGroup>
+        </Box>
+        <TextField
           type="text"
-          placeholder="Search by name and email..."
+          placeholder="Enter name and / or email to find a user."
+          variant='standard'
           value={searchTerm}
           onChange={handleChange}
         />
-        <div className="tab-buttons">
-          <div>
-            <button
-              type="button"
-              className={
-                searchResultType === 'name'
-                  ? 'select-button selected'
-                  : 'select-button'
-              }
-              onClick={buttonSwap}
-              disabled={searchResultType === 'name'}
-            >
-              Results by Name
-            </button>
-          </div>
-          <div>
-            <button
-              type="button"
-              className={
-                searchResultType === 'email'
-                  ? 'select-button selected'
-                  : 'select-button'
-              }
-              onClick={buttonSwap}
-              disabled={searchResultType === 'email'}
-            >
-              Results by Email
-            </button>
-          </div>
-        </div>
-        <div>
-          <div>
-            <ul className="search-results">
-              {searchResults.map((u) => {
-                return (
-                  // eslint-disable-next-line no-underscore-dangle
-                  <li key={`result_${u._id}`}>
-                    <button
-                      className="search-results-button"
-                      type="button"
-                      onClick={() => setUserToEdit(u)}
-                    >
-                      {searchResultType === 'name'
-                        ? `${u.name?.firstName} ${u.name?.lastName} ( ${u.email} )`
-                        : `${u.email} ( ${u.name?.firstName} ${u.name?.lastName} )`}
-                    </button>
-                  </li>
-                );
-              })}
-            </ul>
-          </div>
-        </div>
-      </div>
-    </div>
+        <Box sx={{ 
```

Comment:
This box needs either `width: 1/1` or `width: "100%"` to fill the contents of it's parent.